starter-core/src/test/groovy/io/micronaut/starter/build/BuildTestVerifier.groovy

```diff
@@ -26,4 +26,6 @@ interface BuildTestVerifier {
     boolean hasDependency(String artifactId)
 
     boolean hasTestResourceDependencyWithGroupId(String expectedGroupId)
+
+    boolean hasBuildDependency(String id)
```

Comment:
`hasBuildPlugin`?

Reply:
Also, where is this used?  Do we need to implement it for Maven?

Reply:
I use it in tests in upcoming PRs

Reply:
renamed: 

https://github.com/micronaut-projects/micronaut-starter/pull/1662/commits/d4cb18cfa1567c9651c5d21ddcd0aa824b969ba9

Test using it: 
https://github.com/micronaut-projects/micronaut-starter/pull/1662/commits/76318cae290d489bc85d4721367a855a7fcdd790